commands/__tests__/accounts.test.ts

```diff
@@ -52,22 +51,13 @@ describe('commands/accounts', () => {
       expect(yargs.demandCommand).toHaveBeenCalledWith(1, '');
     });
 
-    it('should support the correct options', () => {
-      accountsCommand.builder(yargs);
-
-      expect(addConfigOptions).toHaveBeenCalledTimes(1);
-      expect(addConfigOptions).toHaveBeenCalledWith(yargs);
-
-      expect(addAccountOptions).toHaveBeenCalledTimes(1);
-      expect(addAccountOptions).toHaveBeenCalledWith(yargs);
-    });
-
     it('should add the correct number of sub commands', () => {
       accountsCommand.builder(yargs);
       expect(yargs.command).toHaveBeenCalledTimes(subcommands.length);
     });
 
     it.each(subcommands)('should attach the %s subcommand', (name, module) => {
+      console.log(name, module);
```

Comment:
Looks like a console log slipped in (unless intentional?)

Reply:
Whoops, left over from debugging a failure